recipes/tarball.rb

```diff
@@ -67,6 +67,19 @@
   not_if { ::File.exist?(node['cassandra']['source_dir']) }
 end
 
+# Ensure that node['cassandra']['source_dir'] node['cassandra']['installation_dir'] and minus leaf directory exists
+[
+  node['cassandra']['source_dir'].split('/')[0..-2].join('/'),
+  node['cassandra']['installation_dir'].split('/')[0..-2].join('/')
+].each do |dir|
+  directory dir do
+    owner node['cassandra']['user']
+    group node['cassandra']['group']
+    recursive true
+    mode 0755
```

Comment:
Correct me if I'm wrong, but we do need the ability to enter the directory and access files within?
`michal.jankowski@<cut>:~$ install -d -m 644 foobar
michal.jankowski@<cut>:~$ ls -ld foobar
drw-r--r-- 2 michal.jankowski <cut> 68 Dec 10 08:56 foobar
michal.jankowski@<cut>:~$ cd foobar/
bash: cd: foobar/: Permission denied
`